tests/workflows/no_graph/workflow.py

```diff
@@ -0,0 +1,7 @@
+from vellum.workflows import BaseWorkflow
+from vellum.workflows.inputs.base import BaseInputs
+from vellum.workflows.state.base import BaseState
```

Comment:
[nit] Let's remove these imports, showing that we don't need generic parameters in the base case